autorag/nodes/passagereranker/run.py

```diff
@@ -35,6 +35,8 @@ def run_passage_reranker_node(modules: List[Callable],
         os.makedirs(node_line_dir)
     project_dir = pathlib.PurePath(node_line_dir).parent.parent
     retrieval_gt = pd.read_parquet(os.path.join(project_dir, "data", "qa.parquet"))['retrieval_gt'].tolist()
+    retrieval_gt = [[str(uuid) for uuid in inner_array[0]] if inner_array[0].size > 0 else [] for inner_array in
+                    retrieval_gt]
```

Comment:
What is the point of this code? 
This code deletes information in the each retrieval_gt, because it uses only first index of `inner_array`. 
It might be work on 'single retrieval_gt' but certainly do not work on multiple retrieval_gt situation.